output/cloud/output_test.go

```diff
@@ -545,21 +488,12 @@ func TestCloudOutputRequireScriptName(t *testing.T) {
 
 func TestCloudOutputAggregationPeriodZeroNoBlock(t *testing.T) {
```

Comment:
I am actually struggling to understand how this test was useful and what it checked even before your changes... :sweat_smile: It originated from https://github.com/grafana/k6/pull/1130, but I think it got progressively mangled with subsequent refactorings :confused: With your latest changes, I am not sure it test anything meaningful beyond the fact that we can start and stop the output, which should already be covered by other tests :confused:

Reply:
Yeah, but AggregationPeriod is not set by default, so presumably this is already covered by a bunch of tests, that's why I thought we can just get rid of it :sweat_smile: anyway, it will go away soon, so it doesn't really matter.

Reply:
I am joining the @na-- with missing the point of the original test and new additions that this PR is bringing to it.

Reply:
Removed 24dfc4d64839a16785d92d53207e14968040187b